gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/engine/db/FabricDatabase.java

```diff
@@ -651,21 +637,7 @@ private void postCreateDDLReplay(final EmbedConnection embedConn,
               ((ReplayableConflatable)qVal).markExecuting();
             }
           }
-          if (maxIterations > 0) {
-            // do not release the lock in the last iteration to block
-            // GfxdDDLMessages and thus avoid missing any DDL messages
-            this.memStore.releaseDDLReplayLock(true);
-            acquiredReplayLock = false;
-          }
-          if ((actualSize = currentQueue.size()) == 0) {
-            // we are good to end; force the next iteration to be the last
-            // one which is still required to take locks etc. and ensure
-            // flush of any pending DDLs/procedures
-            if (maxIterations > 1) {
-              maxIterations = 1;
-            }
-            continue;
-          }
```

Comment:
I will suggest commenting out the old code instead of removing. We don't know if customers may look for previous behaviour.